components/RoadmapPage/RoadmapPage.tsx

```diff
@@ -25,14 +24,16 @@ const RoadmapPage = () => {
   const changeHoverIndex = (newIndex: number) => setHoveredIndex(newIndex)
 
   useEffect(() => {
+    let offset
+
     stages.map((stage: StageData, index: number) => {
       // eslint-disable-next-line react-hooks/exhaustive-deps
       if (index === 0) offset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
 
       const currentOffset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
 
       if (offset > currentOffset) setCurrentIndex(index)
-      offset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
+      offset = currentOffset
```

Comment:
what is `offset` being used for after `line 35`?

can we just delete this line?